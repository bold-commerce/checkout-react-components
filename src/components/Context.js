import React from 'react';

const CheckoutContext = React.createContext();

export default CheckoutContext;
