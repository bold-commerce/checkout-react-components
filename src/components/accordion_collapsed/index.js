// eslint-disable-next-line import/prefer-default-export
export { default as AccordionCollapsed } from './AccordionCollapsed';
