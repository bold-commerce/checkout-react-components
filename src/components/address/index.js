// eslint-disable-next-line import/prefer-default-export
export { default as Address } from './Address';
