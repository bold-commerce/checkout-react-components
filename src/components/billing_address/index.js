// eslint-disable-next-line import/prefer-default-export
export { default as BillingAddress } from './BillingAddress';
