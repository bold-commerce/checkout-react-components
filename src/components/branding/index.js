// eslint-disable-next-line import/prefer-default-export
export { default as Branding } from './Branding';
