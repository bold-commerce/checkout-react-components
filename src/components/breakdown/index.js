// eslint-disable-next-line import/prefer-default-export
export { default as Breakdown } from './Breakdown';
