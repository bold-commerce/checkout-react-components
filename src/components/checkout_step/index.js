export { default as CheckoutStep } from './CheckoutStep';