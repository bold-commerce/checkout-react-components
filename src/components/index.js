export * from './accordian_collapsed';
export * from './accordion';
export * from './address';
export * from './billing_address';
export * from './branding';
export * from './breakdown';
export * from './checkout_app';
export * from './checkout_button';
export * from './customer';
export * from './discount';
export * from './empty_state';
export * from './line_items';
export * from './loading_state';
export * from './order_processed';
export * from './order_processing';
export * from './one_click_summary';
export * from './one_click_shipping_lines_wrapper';
export * from './one_click_email';
export * from './one_click_step_wrapper';
export * from './checkout_step';
export * from './minimized_email';
export * from './minimized_shipping_address';
export * from './minimized_shipping_method';
export * from './one_click_discount';
export * from './one_click_breakdown';
export * from './payment';
export * from './product';
export * from './product_quantity';
export * from './provider';
export * from './shipping_address';
export * from './shipping_lines';
export * from './summary';
export * from './summary_collapsed';
export * from './redacted_credit_card';
