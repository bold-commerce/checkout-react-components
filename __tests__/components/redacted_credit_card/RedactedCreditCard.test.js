import React from 'react';
import { render } from '@testing-library/react';
import { RedactedCreditCard } from '../../../src/components/redacted_credit_card';

describe('RedactedCreditCard', () => {
  test('renders RedactedCreditCard component', () => {
    const { asFragment } = render(
      <RedactedCreditCard
        brand="Visa"
        lineText="1234"
      />,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
