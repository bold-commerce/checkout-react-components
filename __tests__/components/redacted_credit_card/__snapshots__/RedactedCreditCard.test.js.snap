// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RedactedCreditCard renders RedactedCreditCard component 1`] = `
<DocumentFragment>
  <span
    class="RedactedCard"
  >
    <span
      class="RedactedCard__Brand"
    >
      Visa
    </span>
    <span
      class="RedactedCard__Number"
    >
       •••• •••• •••• 1234
    </span>
  </span>
</DocumentFragment>
`;
